ruuvitag: Shell test fails without auto_init

#### Description
When trying to run the shell test on the ruuvitag board it fails, as the shell does not respond. The reason is that auto_init is not being included, so xtimer (which is needed by stdio_rtt) is not initialized.

#### Steps to reproduce the issue
Run the shell test on the ruuvitag (o other board that uses stdio_rtt):
`BOARD=ruuvitag make all flash test -C tests/shell`

#### Expected results
<details><summary>Expected output</summary>
<p>

```
2019-05-29 10:18:43,362 - INFO # main(): This is RIOT! (Version: 2019.07-devel-470-g79c09-pr/tools/jlink_term_fix)
2019-05-29 10:18:43,362 - INFO # test_shell.
> start_test
2019-05-29 10:18:46,677 - INFO #  main(): This is RIOT! (Version: 2019.07-devel-470-g79c09-pr/tools/jlink_term_fix)
2019-05-29 10:18:46,677 - INFO # test_shell.
> 2019-05-29 10:18:46,677 - INFO #  start_test
2019-05-29 10:18:46,677 - INFO # [TEST_START]
> end_test
2019-05-29 10:18:46,758 - INFO #  end_test
2019-05-29 10:18:46,758 - INFO # [TEST_END]
> 

123456789012345678901234567890123456789012345678901234567890
2019-05-29 10:18:46,860 - INFO #  end_test
2019-05-29 10:18:46,860 - INFO # [TEST_END]
> 2019-05-29 10:18:46,911 - INFO #  end_test
2019-05-29 10:18:46,911 - INFO # [TEST_END]
> 2019-05-29 10:18:47,006 - INFO #  123456789012345678901234567890123456789012345678901234567890
2019-05-29 10:18:47,018 - INFO # shell: command not found: 123456789012345678901234567890123456789012345678901234567890
> unknown_command
2019-05-29 10:18:47,116 - INFO #  unknown_command
2019-05-29 10:18:47,116 - INFO # shell: command not found: unknown_command
> help
2019-05-29 10:18:47,211 - INFO #  help
2019-05-29 10:18:47,212 - INFO # Command              Description
2019-05-29 10:18:47,213 - INFO # ---------------------------------------
2019-05-29 10:18:47,213 - INFO # start_test           starts a test
2019-05-29 10:18:47,213 - INFO # end_test             ends a test
2019-05-29 10:18:47,214 - INFO # echo                 prints the input command
2019-05-29 10:18:47,214 - INFO # reboot               Reboot the node
2019-05-29 10:18:47,214 - INFO # ps                   Prints information about running threads.
2019-05-29 10:18:47,232 - INFO # app_metadata         Returns application metadata
> echo a string
2019-05-29 10:18:47,313 - INFO #  echo a string
2019-05-29 10:18:47,314 - INFO # "echo""a""string"
> ps
2019-05-29 10:18:47,414 - INFO #  ps
2019-05-29 10:18:47,414 - INFO # 	pid | state    Q | pri 
2019-05-29 10:18:47,414 - INFO # 	  1 | pending  Q |  15
2019-05-29 10:18:47,414 - INFO # 	  2 | running  Q |   7
> reboot
2019-05-29 10:18:47,515 - INFO #  main(): This is RIOT! (Version: 2019.07-devel-470-g79c09-pr/tools/jlink_term_fix)
2019-05-29 10:18:47,515 - INFO # test_shell.
> 
```
</p>
</details>

#### Actual results
<details><summary>Actual output</summary>
<p>

```
2019-05-29 10:04:36,053 - INFO # main(): This is RIOT! (Version: 2019.07-devel-470-g79c09-pr/tools/jlink_term_fix)
2019-05-29 10:04:36,053 - INFO # test_shell.
> 2019-05-29 10:04:39,456 - INFO #  main(): This is RIOT! (Version: 2019.07-devel-470-g79c09-pr/tools/jlink_term_fix)
2019-05-29 10:04:39,456 - INFO # test_shell.
> start_test
Timeout in expect script at "child.expect_exact(line)" (tests/shell/tests/01-run.py:48)

RIOT/tests/shell/../../Makefile.include:578: recipe for target 'test' failed
make: *** [test] Error 1
```
</p>
</details>

#### Versions
```
Operating System Environment
-----------------------------
       Operating System: "Ubuntu" "18.04.2 LTS (Bionic Beaver)"
                 Kernel: Linux 4.15.0-50-generic x86_64 x86_64

Installed compiler toolchains
-----------------------------
             native gcc: gcc (Ubuntu 7.4.0-1ubuntu1~18.04) 7.4.0
      arm-none-eabi-gcc: arm-none-eabi-gcc (GNU Tools for Arm Embedded Processors 7-2017-q4-major) 7.2.1 20170904 (release) [ARM/embedded-7-branch revision 255204]
                avr-gcc: avr-gcc (GCC) 5.4.0
       mips-mti-elf-gcc: missing
             msp430-gcc: msp430-gcc (GCC) 4.6.3 20120301 (mspgcc LTS 20120406 unpatched)
   riscv-none-embed-gcc: missing
   xtensa-esp32-elf-gcc: xtensa-esp32-elf-gcc (crosstool-NG crosstool-ng-1.22.0-80-g6c4433a) 5.2.0
   xtensa-lx106-elf-gcc: xtensa-lx106-elf-gcc (crosstool-NG crosstool-ng-1.22.0-60-g37b07f6) 4.8.5
                  clang: clang version 6.0.0-1ubuntu2 (tags/RELEASE_600/final)

Installed compiler libs
-----------------------
   arm-none-eabi-newlib: "2.5.0"
    mips-mti-elf-newlib: missing
riscv-none-embed-newlib: missing
xtensa-esp32-elf-newlib: "2.2.0"
xtensa-lx106-elf-newlib: "2.0.0"
               avr-libc: "2.0.0" ("20150208")

Installed development tools
---------------------------
                  cmake: cmake version 3.10.2
               cppcheck: Cppcheck 1.82
                doxygen: 1.8.13
                 flake8: 3.6.0 (mccabe: 0.6.1, pycodestyle: 2.4.0, pyflakes: 2.0.0) CPython 3.6.7 on Linux
                    git: git version 2.17.1
                   make: GNU Make 4.1
                openocd: Open On-Chip Debugger 0.10.0+dev-00523-g2a3b709a (2018-08-23-10:48)
                 python: Python 2.7.15rc1
                python2: Python 2.7.15rc1
                python3: Python 3.6.7
             coccinelle: missing
```
<!-- Thanks for contributing! -->

--- a/tests/shell/main.c
+++ b/tests/shell/main.c
@@ -24,6 +24,10 @@
 #include "shell_commands.h"
 #include "shell.h"
 
+#if MODULE_STDIO_RTT
+#include "xtimer.h"
+#endif
+
 static int print_teststart(int argc, char **argv)
 {
     (void) argc;
@@ -64,6 +68,9 @@ int main(void)
 
     printf("test_shell.\n");
 
+#if MODULE_STDIO_RTT
+    xtimer_init();
+#endif
 
     /* define buffer to be used by the shell */
     char line_buf[SHELL_DEFAULT_BUFSIZE];

